import_activity_events.py

```diff
@@ -173,11 +176,11 @@ def import_events(force_reload=False):
         db.run("COMMIT TRANSACTION")
 
 
-def fixup_event_data(b, week):
+def fixup_event_data(b, day):
     """Download a data file, remove invalid lines, and re-upload"""
-    orig_key = b.get_key(urlparse.urlparse(EVENTS_FILE_URL.format(week=week)).path[1:])
+    orig_key = b.get_key(urlparse.urlparse(EVENTS_FILE_URL.format(day=day)).path[1:])
     assert orig_key is not None
-    fixed_key = b.new_key(urlparse.urlparse(EVENTS_FILE_FIXED_URL.format(week=week)).path[1:])
+    fixed_key = b.new_key(urlparse.urlparse(EVENTS_FILE_FIXED_URL.format(day=day)).path[1:])
```

Comment:
These activity-event changes all look fine, but I suspect we cant run them as-is because it will mix the old and new data together in a single table.  Should we bite the bullet and migrate to a new table (say "activity_events") for the new 100%-sample-rate data, and keep it separate from the old sampled data?  (We could also move off using the union-of-views trick at the same time).